docs/en/stack/ml/df-analytics/dfa-classification.asciidoc

```diff
@@ -64,10 +54,33 @@ otherwise the {classanalysis} may not provide the best predictions. Read
 The ensemble algorithm that we use in the {stack} for {classification} is a type 
```

Comment:
> The ensemble algorithm that we use....

This can likely be addressed in a separate PR, but I think we can rewrite these sentence to take "us" out of the picture.  For example, something like:  Classification analysis uses an ensemble algorithm... It uses decision trees...